narwhal/config/src/lib.rs

```diff
@@ -96,10 +96,6 @@ pub trait Export: Serialize {
 
 impl<S: Serialize> Export for S {}
 
-// TODO: the stake and voting power of a validator can be different so
```

Comment:
I found this comment confusing and asked about it on Slack. Per that discussion, this will always refer to the 10K voting power and so I felt the comment was more obscuring than clarifying. Since this is hard-coded and there are no plans to change 10K value, to me it seemed not worth keeping a TODO. I could put back in a TODO to rename it from `Stake` to `VotingPower` if you like?